toolkit/scripts/pkggen.mk

```diff
@@ -257,7 +257,6 @@ $(STATUS_FLAGS_DIR)/build-rpms.flag: $(preprocessed_file) $(chroot_worker) $(go-
 		--timestamp-file=$(TIMESTAMP_DIR)/scheduler.jsonl \
 		--toolchain-manifest="$(TOOLCHAIN_MANIFEST)" \
 		$(if $(CONFIG_FILE),--base-dir="$(CONFIG_BASE_DIR)") \
-		$(if $(filter y,$(RUN_CHECK)),--run-check) \
```

Comment:
I think this should stay?

Reply:
Ah, crap! I wanted to remove this from the scheduler, not the reader.:/ Thanks for catching this!

Reply:
It's not used anymore.